src/protocols/ssh/input.c

```diff
@@ -31,6 +33,20 @@
 
 #include <pthread.h>
 
+#include <stdio.h>
+#include <sys/timeb.h>
+
+
+int state = 0;
+long long start_1, start_2, start_3 = 0;
+long long end_1, end_2, end_3 = 0;
```

Comment:
Global state should be avoided where possible. This would make more sense within the state representation of the internal terminal struct, and needs to be documented.